src/routes/app/projects/git/[gitUrl]/+page.ts

```diff
@@ -0,0 +1,50 @@
+import type { GenericGitSource, GitProject } from '../../types';
+import type { PageLoad } from './$types';
+
+export const load: PageLoad = ({ params }) => {
+  const { gitUrl } = params;
+
+  const decodedGitUrl = new URL(decodeURI(gitUrl));
+
+  // TODO: Fetch real repo data
```

Comment:
Out of PR context: I guess we'll need helpers for each source, right? Let's discuss this.